linux-anvil-alma/Dockerfile

```
@@ -0,0 +1,92 @@
+ARG DISTRO_ARCH
+ARG DISTRO_NAME
+ARG DISTRO_VER
+FROM --platform=linux/${DISTRO_ARCH} ${DISTRO_NAME}:${DISTRO_VER}
+
+LABEL maintainer="conda-forge <conda-forge@googlegroups.com>"
+
+ARG DISTRO_ARCH
+ARG DISTRO_NAME
+ARG DISTRO_VER
+ENV DISTRO_ARCH=${DISTRO_ARCH} \
+    DISTRO_NAME=${DISTRO_NAME} \
+    DISTRO_VER=${DISTRO_VER}
+
+# Set an encoding to make things work smoothly.
+ENV LANG en_US.UTF-8
+
+# Add qemu in here so that we can use this image on regular linux hosts with qemu user installed
+ADD qemu-aarch64-static /usr/bin/qemu-aarch64-static
+ADD qemu-arm-static /usr/bin/qemu-arm-static
+ADD qemu-ppc64le-static /usr/bin/qemu-ppc64le-static
+ADD qemu-s390x-static /usr/bin/qemu-s390x-static
+
+# bust the docker cache so that we always rerun the installs below
+ADD https://loripsum.net/api /opt/docker/etc/gibberish
+
+# Resolves a nasty NOKEY warning that appears when using yum.
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
+
+# Install basic requirements.
+COPY scripts/yum_clean_all /opt/docker/bin/
+RUN yum update -y && \
+    yum install -y \
+        bzip2 \
+        glibc-langpack-en \
```

We used the minimal lang pack in the sysroot. I think that'd be better here since it includes more languages.

---

Sure we can do that. Something like this?

```suggestion
        glibc-minimal-langpack \
```

Initially was just trying to get us past the locale failures on CI, which this seems to have done. Unlike CentOS 7, this wasn't preinstalled for us (hence the CI failure from before). This seems to have fixed it 🙂

---

Yep that should work!